app/assets/src/components/views/report/SampleDetailsSidebar/constants.js

```diff
@@ -44,6 +47,40 @@ export const METADATA_SECTIONS = [
   }
 ];
 
+// Includes mosquito and other vectors.
+export const VECTOR_METADATA_SECTIONS = [
+  {
+    name: "Sample Info",
+    keys: [
+      "unique_id",
+      "sample_type",
+      "nucleotide_type",
+      "sample_unit",
```

Comment:
What is the "sample unit"? Asking just because it is not part of Sample Info in the human schema. 
I am wondering how this is going to evolve, and if we should try to keep these sections similar  and common. Then you would just enable sections to a human or vector. It would be easier to scale.
Just flagging.. It is not that relevant yet, but if/when we add more we should have the discussion.

Reply:
I am not fond of having the metadata to show defined on both frontend and backend, needing to have to update frontend everytime we add a new field. I fear this might change quite a lot.

Reply:
That's a great point. I originally restrained from adding "sample info" and "donor info" to the back-end because they seemed like display-specific attributes (i.e. for front-end only), but now that I think about it, there's no reason not to.

I will re-think this in a follow-up commit.

Reply:
Sample Unit seems to be whether the sample is comprised of a single organism or pooled from a bunch. Not 100% sure what it is. But it seemed like sample info.